io/pci/pci_hotplug.py

```diff
@@ -46,7 +46,7 @@ def setUp(self):
         """
         Setup the device.
         """
-        if 'ppc' not in platform.processor():
+        if 'ppc' not in cpu.get_arch():
```

Comment:
cpu.get_arch will actually returns `powerpc` and not ppc 

https://github.com/avocado-framework/avocado/blob/3b1f5cbbb07ea850498b55aa020d74cc0f7fa15f/avocado/utils/cpu.py#L200C31-L200C38

Reply:
```
>>> from avocado.utils import cpu
>>> cpu.get_arch()
'ppc64le'
>>> 
```